src/lib/util/TelemetryUtil.ts

```diff
@@ -0,0 +1,11 @@
+import { Lifecycle } from '@salesforce/core';
+import { TelemetryData } from '../../types';
+
+
+export async function emitTelemetry(data: TelemetryData): Promise<void> {
+	try {
+		await Lifecycle.getInstance().emitTelemetry(data);
+	} catch (e) {
+		// Catch-block intentionally left empty to swallow error.
```

Comment:
Can we put a line to the log when this happens? Warning level should make it show up on the logs every time and this will help us investigate if there's an issue.

Reply:
Good point. A log line sounds good, but do we maybe want to go one step further and also make it a verbose-only message to the user? Or is that excessive?

Reply:
Was thinking about verbose info as well - worried if that'll make the scanner command go recursive. If that shouldn't be a problem, agree that info-verbose would be helpful too.

Reply:
I think that should be doable. Stand by.